### PR TITLE
[cppzmq] Fix can't be found with "ZeroMQ"

### PR DESCRIPTION
**Pull request by shirok1**
- #### What does your PR fix?
  Fix that the description of `cppzmq` doesn't include "ZeroMQ", so it can't be found with "ZeroMQ". No code changes included.

### CODE REVIEW

**Comment by Cheney-W** [comment]
Please add `"port-version": 1,` into the vcpkg.json file.

**Comment by Cheney-W** [comment]
Please run below command to format the vcpkg.json:
`.\vcpkg format-manifest ./ports/cppzmq/vcpkg.json`

then run below commands to update the version file of `cppzmq` in `vcpkg/tree/master/versions` folder:
```
.\vcpkg x-add-version cppzmq
git add .
git commit -m "xxxx"
```

**Comment by shirok1** [comment]
@microsoft-github-policy-service agree

**Review by vicroms — DISMISSED** [review]

File: versions/c-/cppzmq.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "versions": [
+    {
+      "git-tree": "b065174fd76bfdebddc5ff71c5b8a7feaf0d80db",
```
[the remark this answers is not in the document]
Comment (shirok1) [reply versions/c-/cppzmq.json:4]: Sorry but I added this after the bot prompting me to add a license for it, so I did it in a seperated commit.

**Comment by BillyONeal** [comment]
Thanks :)

**Pull request merged**